crates/nu-path/src/pwd_per_drive.rs

```diff
@@ -0,0 +1,318 @@
+#[cfg(windows)]
```

Comment:
Not really sure you need this if the whole `pwd_per_drive` module is windows-only. Not sure you need the `_impl` module in general

Reply:
OK, it's redundant condition, should be removed